testng-core/src/main/java/org/testng/internal/YamlParser.java

```diff
@@ -21,6 +21,7 @@ public XmlSuite parse(String filePath, InputStream is, boolean loadClasses)
 
   @Override
   public boolean accept(String fileName) {
-    return Parser.hasFileScheme(fileName) && fileName.endsWith(".yaml");
+    return Parser.hasFileScheme(fileName) && fileName.endsWith(".yaml")
+        || fileName.endsWith(".yml");
```

Comment:
Please, use parentheses between the or operation to help reading.

Reply:
Made requested changes.